samples/deleteSnippets.js

```diff
@@ -0,0 +1,164 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  instanceId = 'YOUR_INSTANCE_ID',
+  tableId = 'YOUR_TABLE_ID',
+  deleteType = 'readRow'
+) {
+  // [START bigtable_deletes_print]
+  const {Bigtable} = require('@google-cloud/bigtable');
+  const bigtable = new Bigtable();
+
+  // TODO: We need to import Mutation.methods.DELETE from mutation.ts, but I guess we need to compile typescript?
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const instanceId = 'YOUR_INSTANCE_ID';
+  // const tableId = 'YOUR_TABLE_ID';
+  const instance = bigtable.instance(instanceId);
+  const table = instance.table(tableId);
+
+  switch (deleteType) {
+    case 'deleteFromColumn': {
+      // [START bigtable_delete_from_column]
+      await table.mutate({
+        key: 'phone#4c410523#20190501',
+        method: 'delete',
+        data: {
+          column: 'cell_plan:data_plan_05gb',
+        },
+      });
+      await printRows();
+      // [END bigtable_delete_from_column]
+      break;
+    }
+    case 'deleteFromFamily': {
+      // [START bigtable_delete_from_column_family]
+      await table.mutate({
+        key: 'phone#4c410523#20190501',
+        method: 'delete',
+        data: {
+          column: 'cell_plan',
+        },
+      });
+      await printRows();
+      // [END bigtable_delete_from_column_family]
+      break;
+    }
+    case 'deleteFromRow': {
+      // [START bigtable_deletes_from_row]
+      const row = table.row('phone#4c410523#20190501');
+      await row.delete();
+      await printRows();
+      // [END bigtable_deletes_from_row]
+      break;
+    }
+    case 'streamingAndBatching': {
+      // [START bigtable_streaming_and_batching]
+      const rows = (await table.getRows({limit: 1}))[0];
+      const TIMESTAMP = new Date(2019, 5, 3);
+      TIMESTAMP.setUTCHours(0);
+      const entries = rows.map(row => {
+        return {
+          key: row.id,
+          method: 'insert',
+          data: {
+            stats_summary: {
+              connected_wifi: {
+                value: 4,
+                labels: [],
+                timestamp: TIMESTAMP,
+              },
+            },
+          },
+        };
+      });
+      await table.mutate(entries);
```

Comment:
Perhaps mutate at least two entries, to demonstrate the "batching" part? 
I noticed that the `writeBatch` sample also mutates 2 rows: https://github.com/googleapis/nodejs-bigtable/blob/e5f6fdb5345a19055be812c59796a2e08ea45345/samples/writeBatch.js#L33

Reply:
Done using the following line of code now:

`const rows = (await table.getRows({limit: 2}))[0];`